### PR TITLE
Send signup message to specific channel

### PR DESCRIPTION
**Pull request by kwunkiuma**
Starting a signup will now send the signup message to the channel named events.

If the channel doesn't exist then the users is informed of the error.

### CODE REVIEW

File: DiscordBot/Commands/EventCommands.cs [hunk 1]
```diff
@@ -381,18 +384,32 @@ private async Task CreateSignupSheet(CommandContext context, int eventKey)
                 return;
             }
 
-            await SendSignupMessage(context, eventKey);
+            try
+            {
+                var signupChannel = context.Guild.Channels
+                    .First(channel => channel.Value.Name == SignupChannelName)
```
Comment (FridaTveit) [inline DiscordBot/Commands/EventCommands.cs:390]: Minor but you could probably use `FirstOrDefault` here instead and avoid the try/catch (instead check for null). Makes the code less nested and a bit easier to read I think 🙂

File: DiscordBot/Commands/EventCommands.cs [hunk 2]
```diff
@@ -17,6 +17,9 @@ namespace DiscordBot.Commands
     internal class EventCommands : BaseCommandModule
     {
         public const string ClearReaction = ":no_entry_sign:";
+        public const string RefreshReaction = ":arrows_counterclockwise:";
+
+        public const string SignupChannelName = "events";
```
Comment (FridaTveit) [inline DiscordBot/Commands/EventCommands.cs:22]: Do these new consts need to be public? 🙂

**Review by FridaTveit — COMMENTED** [review]
A couple of comments, otherwise looks good 🙂

File: DiscordBot/Commands/EventCommands.cs (hunk 2 above)
Comment (kwunkiuma) [reply DiscordBot/Commands/EventCommands.cs]: RefreshReaction should have been used in Program.cs but SignupChannelName can be private.

File: DiscordBot/Commands/EventCommands.cs (hunk 1 above)
Comment (kwunkiuma) [reply DiscordBot/Commands/EventCommands.cs]: Done!

File: DiscordBot/Commands/EventCommands.cs [hunk 3]
```diff
@@ -384,18 +384,17 @@ private async Task CreateSignupSheet(CommandContext context, int eventKey)
...
             }
 
-            try
-            {
-                var signupChannel = context.Guild.Channels
-                    .First(channel => channel.Value.Name == SignupChannelName)
-                    .Value;
+            var signupChannel = context.Guild.Channels
+                .FirstOrDefault(channel => channel.Value.Name == SignupChannelName)
+                .Value;
```
Comment (FridaTveit) [inline DiscordBot/Commands/EventCommands.cs:389]: I don't think you can call `Value` here if null has been returned? Have you tested this for a channel which doesn't exist? 🙂

**Review by FridaTveit — COMMENTED** [review]
One question otherwise looks good 🙂

File: DiscordBot/Commands/EventCommands.cs (hunk 3 above)
Comment (kwunkiuma) [reply DiscordBot/Commands/EventCommands.cs:389]: I had that same thought and wanted to use ?.Value but the IDE didn't allow it and it appears to be working when the channel doesn't exist.
image

Comment (kwunkiuma) [reply DiscordBot/Commands/EventCommands.cs:389]: It seems the default value is a key value pair of [0, null].

Comment (FridaTveit) [reply DiscordBot/Commands/EventCommands.cs:389]: Ah I see 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**